internal/controllers/reconciliation/crud_test.go

```diff
@@ -861,7 +861,9 @@ func TestResourceDefaulting(t *testing.T) {
 									"name":  "foo",
 									"image": "bar",
 									"resources": map[string]any{
-										"memory": "1024Mi", // apiserver will return this as "1Gi"
+										"limits": map[string]any{
```

Comment:
Huh, memory is not even a valid field of the resources key right? How did it ever pass...

Reply:
Eno allows unknown properties so that should explain why it passed, I'm just not sure it could have failed lol